lib/c-api/src/wasm_c_api/store.rs

```diff
@@ -0,0 +1,31 @@
+use super::engine::wasm_engine_t;
+use std::ptr::NonNull;
+use wasmer::Store;
+
+/// Opaque wrapper around `Store`
+/// cbindgen:ignore
+#[allow(non_camel_case_types)]
+pub struct wasm_store_t {}
+
+/// cbindgen:ignore
```

Comment:
Is there a reason we're putting `/// cibndgen:ignore` on everything inside a module instead of at the module level?

Reply:
Nop. Updated :-).